src/libcommonserver/utility/utility.cpp

```diff
@@ -390,10 +390,29 @@ bool Utility::endsWithInsensitive(const std::string &str, const std::string &suf
                       [](char c1, char c2) { return std::tolower(c1, std::locale()) == std::tolower(c2, std::locale()); });
 }
 
-bool Utility::isEqualInsensitive(const std::string &strA, const std::string &strB) {
-    return strA.size() == strB.size() && std::equal(strA.begin(), strA.end(), strB.begin(), strB.end(), [](char a, char b) {
-               return std::tolower(a, std::locale()) == std::tolower(b, std::locale());
-           });
+bool Utility::isEqualUpToCaseAndEnc(const SyncPath &a, const SyncPath &b, bool &isEqual) {
```

Comment:
```suggestion
bool Utility::isEqualUpToCaseAndEncoding(const SyncPath &a, const SyncPath &b, bool &isEqual) {
```
We should avoid abbreviation that are not commonly used in the English language or coding community

Reply:
What about `checkIfCaseAndEncodingAreEqual`?

Reply:
Renaming done here : f89eb2b7ae943668551302884673a604d18e8e07.